### PR TITLE
2x_fixes

### LINKED ISSUE #1316: Sidebar buttons overlay drop-down menus in panels
It is not obvious with English locale, but when you switch a language, the components get different widths and the problem is evident then.
image

### LINKED ISSUE #1317: Composition keywords display [object Object] instead of keywords
image

### PR DESCRIPTION
**Pull request by FilipLeitner**
- add name param to composition schema, necessary since layman itself started to distinguish between name and title. In case the two values are different loading a layer from layman would fail
- Fix loading of two or more datasources catalogues of the same type
- Add warning for user trying to save drawings/ create drawing layer and is not logged in layman
- Fix advanced micka search component
- Partialy fix suggestion micka component
- Fix #1316 Sidebar buttons overlay drop-down menus
- Fix #1317 Composition keywords display [object Object] instead of keywords
- Fix compositions pagination control
- Fix main panel switch to layermanager when trying to add WMS/WFS through datasources

### CODE REVIEW

**Review by jmacura — CHANGES_REQUESTED** [review]

File: components/datasource-selector/micka/micka-filters.service.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ export class HsMickaFilterService {
   suggestionsLoaded = true;
   filterByExtent = true;
   otnKeywords = [];
+  suggestionsModalVisible: boolean;
```
[the remark this answers is not in the document]
Comment (FilipLeitner) [reply components/datasource-selector/micka/micka-filters.service.ts:20]: I guess I wanted to minimize number of dom modifications. So if youd once created micka "advanced search" modal you dont have to remove and recreate it the second time you opening it. Same applies to "suggestions" modal. Service was way to go.
But if we prefer the second approach of add/remove I can just quickly change those

[the remark this answers is not in the document]
Comment (jmacura) [reply components/datasource-selector/micka/micka-filters.service.ts:20]: Ack. If you expect it to be faster, then let's keep it that way.

File: components/compositions/compositions.service.ts [hunk 2]
```diff
@@ -87,6 +87,7 @@ export class HsCompositionsService {
       bbox,
       this.HsCompositionsMapService.extentLayer
     );
+    console.log(ds)
```
Comment (jmacura) [inline components/compositions/compositions.service.ts:90]:
```suggestion
```

**Review by jmacura — CHANGES_REQUESTED** [review]
A tremendous amount of errors in my console, but... yeah... from the user point of view it all works 🙂 

Please, correct the typo and the `log` and it's good to go 👍

**Review by jmacura — APPROVED** [review]
